src/ContentRepository/RepositoryInstance.cs

```diff
@@ -346,11 +346,33 @@ private static void InitializeOAuthProviders()
 
         private static void InitializeLogger()
         {
-            var logSection = ConfigurationManager.GetSection("loggingConfiguration");
-            if (logSection != null)
-                SnLog.Instance = new EntLibLoggerAdapter();
+            // look for the configured logger
+            var loggers = Providers.Instance.GetProvider<IEventLogger[]>();
+            if (loggers?.Length > 0)
+            {
+                //TODO: change this when the logger api can handle multiple loggers
+                SnLog.Instance = loggers[0];
+            }
             else
-                SnLog.Instance = new DebugWriteLoggerAdapter();
+            {
+                // fallback to the builtin logger
+                var logSection = ConfigurationManager.GetSection("loggingConfiguration");
+                if (logSection != null)
+                    SnLog.Instance = new EntLibLoggerAdapter();
+                else
+                    SnLog.Instance = new DebugWriteLoggerAdapter();
+            }
+
+            //set configured tracers
+            var tracers = Providers.Instance.GetProvider<ISnTracer[]>();
+            if (tracers?.Length > 0)
+                SnTrace.SnTracers.AddRange(tracers);

```

Comment:
Need to delete SnTrace.Sntracers list before setting it up because it contains a default tracer.